### DISCUSSION TITLE
Netbox in Platform One

### DISCUSSION THREAD

**Discussion by lvangink** [netbox-community/netbox#10474] [Ideas]
Hi everyone. We are working on building a NetDevOps pipeline for some Government customers and NetBox is our database of choice. We are submitting all the tooling to Platform One, which is the D.o.D. hardened container registry. This essentially gives containers security accreditation across D.o.D. networks. 

My question is if there are any contacts working with NetBox that would like to help with the hardening and submission? 

Details are here - https://repo1.dso.mil/dsop/dccscr

**Comment by abhi1693** [comment]
The link is not working for me

**Reply by lvangink** [reply]
Interesting. It's a public site so you don't need DoD credentials to login. Are you trying to access from outside of the US? Maybe they have IP restrictions since its Government.

**Reply by joaolucasmacedo** [reply]
Probably. I can't access either. I'm in Brazil.
